lanelet2_projection/include/lanelet2_projection/LocalCartesian.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include <lanelet2_io/Exceptions.h>
+#include <lanelet2_io/Projection.h>
+
+#include <GeographicLib/Geocentric.hpp>
+#include <GeographicLib/LocalCartesian.hpp>
+
+namespace lanelet {
+namespace projection {
+class LocalCartesianProjector : public Projector {
+ public:
+  explicit LocalCartesianProjector(Origin origin);
+
+  BasicPoint3d forward(const GPSPoint& gps) const override;
+
+  GPSPoint reverse(const BasicPoint3d& enu) const override;
+
+ private:
+  GeographicLib::LocalCartesian localCartesian;
```

Comment:
should be `localCartesian_`, to conform to our convention for private members

Reply:
renamed